### PR TITLE
feat: add Patreon lists

### PR DESCRIPTION
**Pull request by MattLish**
- Fix UI to match the design for patrons
- Integrate with the real Patreon (via ultsky.phinocio.com/api/patreon)
- Add font variables
- Add reusable Page component
- Add reusable PageContent component
- Add library for using @Prop decorator
- Add basic IoC container
- Add Patreon service

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
-};
+})
+export default class App extends Vue {
+  async created() {
+    registerServices();
+  }
+}
 </script>
 
 <style lang="scss">
```
Comment (phinocio) [inline src/App.vue:35]: Imports should be consistent. font-face as you said could probably just be in settings, though

**Review by phinocio — COMMENTED** [review]
Building produces vertical and horizontal bars for me for some reason. 

font-face should probably be in settings

File: src/App.vue (hunk 1 above)
Comment (MattLish) [reply src/App.vue:35]: Good spot, this should be resolved now and the font faces should be loaded in as part of the settings.

Comment (MattLish) [reply src/App.vue:35]: The issue with the scrolling should have also been solved with a combination of the recent changes and https://github.com/UltimateSkyrim/ultskylauncher/pull/45

Comment (phinocio) [reply src/App.vue:35]: looks good to me 👌

**Review by phinocio — APPROVED** [review]
👌

**Pull request merged**